go/libraries/doltcore/schema/typeinfo/varbinary.go

```diff
@@ -85,6 +85,20 @@ func (ti *varBinaryType) ConvertNomsValueToValue(v types.Value) (interface{}, er
 	return nil, fmt.Errorf(`"%v" cannot convert NomsKind "%v" to a value`, ti.String(), v.Kind())
 }
 
+// ReadFrom reads a go value from a noms types.CodecReader directly
+func (ti *varBinaryType) ReadFrom(_ *types.NomsBinFormat, reader types.CodecReader) (interface{}, error) {
+	k := reader.ReadKind()
```

Comment:
Panic here for now as I've changed all of this logic, and I don't wanna accidentally forget to make the change on my PR.